src/Avalonia.Base/Utilities/CharacterReader.cs

```diff
@@ -46,6 +46,20 @@ public bool TakeIf(char c)
             }
         }
 
+        public bool TakeIf(string s)
```

Comment:
New public API, is that intended? (I don't think CharacterReader should have been exposed in the first place.)

Reply:
Gah, no that wasn't intended. I didn't expect that class to be public!

Reply:
Made this internal, and added a `TODO12` comment to remind us to make this type internal.